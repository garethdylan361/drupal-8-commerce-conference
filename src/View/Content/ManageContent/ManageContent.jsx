import React from 'react';
import Panel from 'react-bootstrap/lib/Panel';
import Row from 'react-bootstrap/lib/Row';
import Col from 'react-bootstrap/lib/Col';

export default class ManageContent extends React.Component {

  constructor(props){
    super(props);
    props.title
  }

  render() {
    return (
      <Panel header="Header">

      </Panel>
    );
  }

}
