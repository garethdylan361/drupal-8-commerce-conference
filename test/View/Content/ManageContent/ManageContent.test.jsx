import React from 'react';
import renderer from 'react-test-renderer';
import ManageContent from '../../../../src/View/Content/ManageContent/ManageContent';

test('ManageContent renders correctly', () => {
  const title = "Conference 2017"
  const id = 1;
  const manage = renderer.create(<ManageContent id={1} title={title} />);
  expect(manage.toJSON()).toMatchSnapshot()
});
