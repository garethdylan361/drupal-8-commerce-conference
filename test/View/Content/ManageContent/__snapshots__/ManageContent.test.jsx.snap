// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ManageContent renders correctly 1`] = `
<div
  className="panel panel-default"
  id={undefined}
>
  <div
    className="panel-heading"
  >
    Header
  </div>
</div>
`;
